var/spack/repos/builtin/packages/gotcha/package.py

```diff
@@ -21,6 +21,7 @@ class Gotcha(CMakePackage):
 
     variant('test', default=False, description='Build tests for Gotcha')
     patch('arm.patch', when='@1.0.2')
+    patch('arm.patch', when='@0.0.2')
```

Comment:
The fix for this issue is at https://github.com/LLNL/GOTCHA/commit/e82b4a1ecb634075d8f5334b796c888c86da0427 and it seems different from this patch. I suspect that the patched version will only work on aarch64 and not on other archs. Can you:
1. Remove the file patch from Spack repo
2. Use https://github.com/LLNL/GOTCHA/commit/e82b4a1ecb634075d8f5334b796c888c86da0427.patch as the url for the patch

?

Reply:
I agree with you.
I will make the changes as you advise.